.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 package-lock.json
-cookie.txt
\ No newline at end of file
+cookie.txt
```

Comment:
I'm wondering if this is necessary. Since everyone is working using different environments, you may just use `git add filename1 filename2 filename 3` to stage the specific files that you want to commit :)

Reply:
@brendajerop I didn't use .gitignore and I have not made any changes in that.

Reply:
ok! I got my mistake and I have modified .gitignore